agt/coco.py

```diff
@@ -7,7 +7,11 @@
 async def emit_responses(state: ConversationState, component_response: CoCoResponse):
     if hasattr(component_response, "responses") and component_response.responses:
         for resp in component_response.responses:
-            await state.say(text=resp.get("text"), image_url=resp.get("image"))
+            await state.say(
+                text=resp.get("text"),
+                image_url=resp.get("image"),
+                ssml=resp.get("ssml") or "",
```

Comment:
why empty string

Reply:
Did not want it to be None at the end of the chain. Can cause exceptions.